play-json/shared/src/main/scala/play/api/libs/json/Writes.scala

```diff
@@ -265,15 +269,48 @@ trait DefaultWrites extends LowPriorityWrites {
   /**
    * Serializer for Map[String,V] types.
    */
+  @deprecated("Use `genericMapWrites`", "2.7.0")
```

Comment:
I don't think it is possible to backport all of these changes to 2.7.x branch, so better to update the deprecation notices to `2.8.0`. I won't highlight them all to avoid being noisy here.

Reply:
Due to delayed review, this PR was there before 2.7.0

Reply:
Yeah, sorry about the delayed review. 

Anyway, since we don't depreciate methods in patch releases, now this should be 2.8.0.

Reply:
up